src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotNew = {};
+
+  for (const key in robot) {
+    if (robotNew[robot[key]]) {
+      return null;
+    }
+
+    if (!robotNew[robot[key]]) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 2nd item, what if value contains falsy value? empty string, null, false, etc. Try to fix your check